pkg/benchmarks/browser.js

```diff
@@ -71,10 +70,19 @@ $("#run").click(() => {
 
     Runner.run(benchmarks, runCount, options, {
 
-        readFile(file) {
+        readBenchmark(file) {
+            return $.ajax({
+                type: "GET",
+                url: `https://raw.githubusercontent.com/peggyjs/peggy/${BRANCH}/examples/${file}`,
```

Comment:
benchmarks should not be dependant on network access to external files from a third party, especially not one with an outage history like github's.

this prevents people on corporate networks from participation.  historically this would have destroyed about 40% of peg's contributions from third parties.

unless the goal is to enforce that the public-facing examples in the repo run, or something?

but ... shouldn't that just be on the local copies?  that means that the local test set enforces a different version of the source than you're actually running it against

Reply:
Nod, this is what the old benchmarks did.  Let's start a separate bug to fix this.

Reply:
Oh okay.  I hadn't looked at them before.

[Made](https://github.com/peggyjs/peggy/issues/78)